### ISSUE TITLE
make bootstrap_sorcha_data_files give more informative output to the terminal 

### ISSUE THREAD

**Issue by mschwamb** [dirac-institute/sorcha#610]
- If the files already exist and the -f flag is not set, output a message 
- add a message if file download is successful is kind of unclear if you know it succeeded in what it is doing because the bars are all going at the same times. 
- ensure if one of the files does not download that an error message is printed out

**Comment by drewoldag** [comment]
**Currently the output from boot strapping looks like this when some files are missing:**
```
(sorcha) drew@pterodactyl sorcha % bootstrap_sorcha_data_files
Checking cache for existing files.
The following file(s) were not found in the local cache: /Users/drew/Library/Caches/sorcha/
- ObsCodes.json
- pck00010.pck
Downloading file 'pck00010.pck' from 'https://naif.jpl.nasa.gov/pub/naif/generic_kernels/pck/pck00010.tpc' to '/Users/drew/Library/Caches/sorcha'.
Decompressing '/Users/drew/Library/Caches/sorcha/ObsCodes.json.gz' to '/Users/drew/Library/Caches/sorcha/ObsCodes.json' using method 'auto'.
100%|████████████████████████████████████████| 126k/126k [00:00<00:00, 132MB/s]
Confirming all expected files were downloaded and created.
All expected files were found in the local cache: /Users/drew/Library/Caches/sorcha/
```

**And looks like this when all files are in the local cache:**
```
(sorcha) drew@pterodactyl sorcha % bootstrap_sorcha_data_files
Checking cache for existing files.
All expected files were found in the local cache: /Users/drew/Library/Caches/sorcha/
```

**When a file cannot be downloaded, the output looks like the following.** I simulated this by removing the local file from the cache and then changing the URL to be refer to a nonexistent file.
```
(sorcha) drew@pterodactyl sorcha % bootstrap_sorcha_data_files
Checking cache for existing files.
The following file(s) were not found in the local cache: /Users/drew/Library/Caches/sorcha/
- pck00010.pck
Downloading file 'pck00010.pck' from 'https://naif.jpl.nasa.gov/pub/naif/generic_kernels/pck/pck00010.tp' to '/Users/drew/Library/Caches/sorcha'.
Failed to download 'pck00010.pck'. Will attempt the download again 1 more time.
Downloading file 'pck00010.pck' from 'https://naif.jpl.nasa.gov/pub/naif/generic_kernels/pck/pck00010.tp' to '/Users/drew/Library/Caches/sorcha'.
Failed to download 'pck00010.pck'. Will attempt the download again 1 more time.
Traceback (most recent call last):
  File "/Users/drew/opt/miniconda3/envs/sorcha/bin/bootstrap_sorcha_data_files", line 8, in <module>
    sys.exit(main())
  File "/Users/drew/code/sorcha/src/sorcha/utilities/retrieve_ephemeris_data_files.py", line 117, in main
    build_meta_kernel_file(retriever)
  File "/Users/drew/code/sorcha/src/sorcha/utilities/generate_meta_kernel.py", line 51, in build_meta_kernel_file
    shortened_file_name = _build_file_name(retriever.abspath, retriever.fetch(file_name))
  File "/Users/drew/opt/miniconda3/envs/sorcha/lib/python3.10/site-packages/pooch/core.py", line 588, in fetch
    stream_download(
  File "/Users/drew/opt/miniconda3/envs/sorcha/lib/python3.10/site-packages/pooch/core.py", line 803, in stream_download
    downloader(url, tmp, pooch)
  File "/Users/drew/opt/miniconda3/envs/sorcha/lib/python3.10/site-packages/pooch/downloaders.py", line 207, in __call__
    response.raise_for_status()
  File "/Users/drew/opt/miniconda3/envs/sorcha/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://naif.jpl.nasa.gov/pub/naif/generic_kernels/pck/pck00010.tp
```

**Comment by mschwamb** [comment]
Addressed by #646
